packages/babel/src/transformation/file/index.js

```diff
@@ -578,7 +591,7 @@ export default class File {
   }
 
   /**
-   * [Please add a description.]
+   * If this should be ignored, pass it through, otherwise run callback.
```

Comment:
The "pass it through" terminology in this comment and the one for `shouldIgnore` seems rather confusing as it's not entirely clear what it means.